apps/server/src/modules/learnroom/controller/rooms.controller.ts

```diff
@@ -53,6 +55,7 @@ export class RoomsController {
 	}
 
 	@Post(':roomid/copy')
+	@Timeout(serverConfig().INCOMING_REQUEST_TIMEOUT_COPY_API)
```

Comment:
@SevenWaysDP  @BjoernDataport Did you test the time out in a staging environment? Is everything working properly with the env vars there?

Reply:
yes, is working